src/decorators/withResources/Resource.js

```diff
@@ -23,6 +25,12 @@ function overrideThen(resource, replaceWith) {
   return resource
 }
 
+function isAllowed(instance, ...to_test) {
```

Comment:
While the usages here are all covered, I think this signature is a bit misleading; calling `isAllowed(this, 'GET', 'POST')` looks supported, but may lead to unexpected results (with `allowed_methods` set to `['GET']`, `isAllowed(this, 'GET', 'POST')` will return `true` (not sure if that's expected)

Maybe using `isAllowed(instance, method)` as signature could make it more clear? Or if it was meant to support multiple methods, what should be the outcome?

Reply:
The idea was to allow for multiple methods as well so that `isAllowed(instance, GET)` and `isAllowed(instance, GET, POST)` would work. So basically if it is allowed to GET and POST. But since only 1 form is used I'll change the signature for now.